src/main/java/net/dv8tion/jda/core/events/message/MessageReceivedEvent.java

```diff
@@ -21,8 +21,10 @@
 
 /**
  * <b><u>MessageReceivedEvent</u></b><br>
- * Fired if a Message is sent in a {@link net.dv8tion.jda.core.entities.MessageChannel MessageChannel}.<br>
- * <br>
+ * Fired if a Message is sent in a {@link net.dv8tion.jda.core.entities.MessageChannel MessageChannel}.
```

Comment:
Due to the changes coming in the Message rewrite, this should probably be "if a Message is received..."

Reply:
Received vs sent?

Reply:
Considering the event is called MessageReceived, should probably keep that in line.

Additionally, we are bringing to JDA a slight separation between the constructs that represent messages that are Sent vs those that are Received.

Reply:
Yea gonna rename every "sent" occurence to "received"